### PR TITLE
Elastic Load Balancer support

### PR DESCRIPTION
**Pull request by egor045**
ELB support:
- Add resources.py to provide ELB class
- Add elbs.py to views/
- Add update_elbs() to region.py

### CODE REVIEW

File: cinq_collector_aws/region.py [hunk 1]
```diff
@@ -496,3 +498,114 @@ def update_vpcs(self):
...
+                        resource_id,
+                        account_id=self.account.account_id,
+                        location=self.region,
+                        properties=properties,
+                        tags=tags
+                    )
+
+                    # elbs[elb.resource.resource_id] = elb
+                    self.log.info(
+                        'Added new EC2Instance {}/{}/{}'.format(
```
Comment (TheM0ng00se) [inline cinq_collector_aws/region.py]: Do you mean ELB instead of E2Instance here?

File: cinq_collector_aws/views/elbs.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            order=4
+        )
+    ]
+
+    @rollback
+    @check_auth(ROLE_USER)
+    def get(self):
+        self.reqparse.add_argument('page', type=int, default=1)
+        self.reqparse.add_argument('count', type=int, default=20, choices=[10, 20, 50])
```
Comment (TheM0ng00se) [inline cinq_collector_aws/views/elbs.py]: I think the counts here feel a bit low? Unless you have specific reasons I'd recommend sticking to the count : 100 in args and choices of 25, 50, 100 to match the other resources

**Review by TheM0ng00se — CHANGES_REQUESTED** [review]
See inline comments, other than that looks good, thanks for this.

**Comment by egor045** [comment]
Log message updated
Counts updated to [25, 50, 100]

**Review by TheM0ng00se — APPROVED** [review]

**Pull request merged**